wgsl/index.bs

```diff
@@ -925,8 +925,10 @@ An attribute [=shader-creation error|must not=] be specified more than once per
     Each parameter [=shader-creation error|must=] be a [=const-expression=] or an [=override-expression=].
     All parameters [=shader-creation error|must=] be the same type, either [=i32=] or [=u32=].
 
-    It is a [=pipeline-creation error=] unless each specified parameter is at least 1 and at most an upper
-    bound specified by the WebGPU API.
+    A [=pipeline-creation error=] results if any specified parameter is 0 or exceeds an upper bound
+    specified by the WebGPU API, or if the product of the parameter values exceeds the
+    [[WebGPU#dom-supported-limits-maxcomputeinvocationsperworkgroup|maxComputeInvocationsPerWorkgroup]] device limit.
+
```

Comment:
Slightly odd that we name this particular limit, but not the dimension ones.  How about:
```suggestion
    A [=pipeline-creation error=] results if any specified parameter is 0 or exceeds an upper bound
    specified by the WebGPU API, or if the product of the parameter values exceeds the
    upper bound specified by the WebGPU API (see [[WebGPU#limits]]).
```

Reply:
I took your suggestion.